typing/env.ml

```diff
@@ -4578,6 +4578,7 @@ let report_lookup_error _loc env ppf = function
         | Error (Linearity, _) -> "once", "is many"
         | Error (Portability, _) -> "nonportable", "is portable"
         | Error (Yielding, _) -> "yielding", "may not yield"
+        | Error (Statefulness, _) -> "stateful", "may not capture state"
```

Comment:
I think that `"is stateless"` might be better here. "may not" can be read as "might not" rather "is not permitted to" and "capture" is probably not going to be understood by a lot of programmers. It could be "is not permitted to access mutable state", but at that point I suspect that "is stateless" is just as good.